### DISCUSSION TITLE
With mirror.exe is it possible to include the share credentials

### DISCUSSION THREAD

**Discussion by RubenMeneses** [dokan-dev/dokany#1203] [Q&A]
Hi
I am trying to use the mirror sample app to try and map and access an SMB file share, is it possible to include the SMB share credentials in the command line? My current command looks like this.
mirror.exe /r "\\\remoteserver\myhomeshare" /l f
Cheers

**Comment by Liryna** [comment]
Hi @RubenMeneses , unfortunately no this is not supported by the mirror sample. If someone is interested, I would be happy to review a pull request that adds this.

**Reply by HitMannCU** [reply]
> Hi @RubenMeneses , unfortunately no this is not supported by the mirror sample. If someone is interested, I would be happy to review a pull request that adds this.

Please!!
